### PR TITLE
Feature/docker healthcheck

### PR DESCRIPTION
**Pull request by bitofant**
added nodejs based health check for docker container health management. Healthcheck.js contains commented hints on how to extend status page parsing. The current implementation is based solely on the http status code.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -19,6 +19,8 @@ RUN npm install --production
 
 COPY . /src
 
+HEALTHCHECK --interval=30s --timeout=30s --start-period=5s --retries=3 CMD node healthcheck.js || exit 1
```
Comment (MartinLang1) [inline Dockerfile]: same comment as in nmap

File: healthcheck.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+	response.on ('data', chunk => {
+		dataChunks.push (chunk);
+	});
+
+	response.on ('close', () => {
+		const txtResponse = Buffer.concat (dataChunks).toString ('utf8');
+		const jsonResponse = JSON.parse (txtResponse);
+		if (jsonResponse.healthcheck === 'DOWN') process.exit (1);
+	});
+	*/
```
Comment (MartinLang1) [inline healthcheck.js]: same comment as nmap pr

**Review by MartinLang1 — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (bitofant) [reply Dockerfile]: All scanners now have a 5s interval

File: healthcheck.js (hunk 2 above)
Comment (bitofant) [reply healthcheck.js]: improved this file

**Pull request merged**